spec/csv_importer_spec.rb

```diff
@@ -372,6 +372,12 @@ class ImportUserCSVByFirstName
     expect { import.run! }.to_not raise_error
   end
 
+  it "can change quote_char value" do
+    import = ImportUserCSV.new(content: "", quote_char: "\x00")
+
+    expect(import.csv.quote_char).to eq "\x00"
+  end
+
```

Comment:
You're right, it would be better to test that things are properly wired up. This file contains high level specs where we test the behaviour of CSVImporter instead of the internals. You could something similar to https://github.com/BrewhouseTeam/csv-importer/pull/26/files#diff-699b1882c566f8988f2b16f89a4bda93R336

```ruby
import = ImportUserCSV.new(content: %|first_name,last_name\nbob "the" second,last_name|, quote_char: "\x00")

import.run!

# assert first imported model attributes here
```